### DISCUSSION TITLE
Retroactively determine which players played on which days/timeslots

### DISCUSSION THREAD

**Discussion by Zweatherly360** [cwendt94/espn-api#377] [ESPN API] [answered]
First of all, I'm loving this API, it's really great! The people in my league really love the advanced analytics. 

I am trying to figure out if it is possible to find out which day (or time slot) each player played in a previous week. I can see it on the ESPN site for future performances (e.g., in week 5, Jalen Hurts plays at 4:25EST vs Arizona), but I can't find it in the web response for either past or future performances. The gist of what I'm trying to do is go back and see how many times each fantasy team in my league had a lead going into Sunday/Monday night and then blew the lead. If there's an easier way to do that, I'd be happy to hear that as well!

Thanks in advance!

**Comment by cwendt94** [comment, marked as answer]
Yes, I am currently calculating in the box_player class if the game is already played. I can add a new field game_date for the box_player so that you can get the date the player played/ is playing.

**Reply by cwendt94** [reply]
You can now get the game date from the `box_scores` function call.

This will get box scores for week 2 and then date of the game that player played.
example: 
```python3
box = league.box_scores(2)
box[0].home_lineup[0].game_date
```

**Reply by Zweatherly360** [reply]
Awesome, thank you so much! This is exactly what I was looking for!
